apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -65,20 +96,598 @@ import { Drawer } from '@harnessio/ui/components'
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur at sagittis justo.
+            Integer vehicula turpis at justo gravida, at convallis lorem tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur at sagittis justo.
+            Integer vehicula turpis at justo gravida, at convallis lorem tincidunt.
+          </p>
+          <p>
+            Suspendisse potenti. Praesent in vestibulum lacus. Cras feugiat eros at ex cursus,
+            vel consequat enim mattis. Quisque nec magna ac leo tempor scelerisque.
+          </p>
+          <p>
+            Nullam tristique, justo vitae laoreet pretium, nisl nunc facilisis nulla, non porta erat sapien in justo.
+            Sed nec massa nec turpis interdum volutpat. Morbi sed justo ac leo sagittis tincidunt.
+          </p>
+        </Drawer.Body>
+
+        <Drawer.Footer>
+          <ButtonGroup>
+            <Button>Action</Button>
+          </ButtonGroup>
+        </Drawer.Footer>
+      </Drawer.Content>
+    </Drawer.Root>
+
+`}
+/>
+
+## Drawer sizes
+
+The `Drawer.Content` component accepts a `size` prop to control its width (or height, depending on `direction`). Supported values are: `sm`, `md`, and `lg`.
+
+<DocsPage.ComponentExample
+  client:only
+  code={`
+    <Drawer.Root>
+      <Drawer.Trigger asChild>
+        <Button>Open Drawer with 'md' size</Button>
+      </Drawer.Trigger>
+
+      <Drawer.Content size="md">
+        <Drawer.Header>
+          <Drawer.Title>Title</Drawer.Title>
+        </Drawer.Header>
+
+        <Drawer.Body />
+
+        <Drawer.Footer>
+          <ButtonGroup>
+            <Drawer.Close asChild>
+              <Button variant="outline">Cancel</Button>
+            </Drawer.Close>
+          </ButtonGroup>
+        </Drawer.Footer>
+      </Drawer.Content>
+    </Drawer.Root>
+
+`}
+/>
+
+<DocsPage.ComponentExample
```

Comment:
I wonder if we should combine these examples into one block? The code won't be as easy to read, but the interaction will be nicer 🤔

Reply:
Either way, can we add a `sm` version too?